crawlers/mooncrawl/mooncrawl/leaderboards_generator/cli.py

```diff
@@ -0,0 +1,206 @@
+import argparse
+import json
+import logging
+import os
+from typing import Any, Dict
+import uuid
+
+import requests  # type: ignore
+
+
+from .utils import get_results_for_moonstream_query
+from ..settings import (
+    MOONSTREAM_ADMIN_ACCESS_TOKEN,
+    MOONSTREAM_LEADERBOARD_GENERATOR_JOURNAL_ID,
+    BUGOUT_REQUEST_TIMEOUT_SECONDS,
+)
+
+from ..settings import bugout_client as bc
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def handle_leaderboards(args: argparse.Namespace) -> None:
+    """
+    Run the leaderboard generator.
+
+    Get query from journal and push results to leaderboard API.
+    """
+
+    ### get leaderboard journal
+
+    query = "#leaderboard"
+
+    if args.leaderboard_id:
+        query += f" #cleaderboard_id:{args.leaderboard_id}"
+
+    leaderboards = bc.search(
```

Comment:
I suggest it to put in try except block with return from function so we will handle errors if spire down and do not respond correctly